projects/skia/skia.diff

```diff
@@ -360,6 +360,45 @@ index 48954a8938..5a4f354a21 100644
      Builder builder;
      for (int i = 0; i < count; i++) {
          LayerInfo info;
+diff --git a/src/gpu/GrPathUtils.cpp b/src/gpu/GrPathUtils.cpp
+index 74a53e9e09..872b28ced6 100644
+--- a/src/gpu/GrPathUtils.cpp
++++ b/src/gpu/GrPathUtils.cpp
+@@ -394,7 +394,11 @@ void convert_noninflect_cubic_to_quads(const SkPoint p[4],
+                                        SkPathPriv::FirstDirection dir,
+                                        SkTArray<SkPoint, true>* quads,
+                                        int sublevel = 0) {
+-
++    #if defined(IS_FUZZING)
++        if (sublevel >= 7) {
```

Comment:
I obtained this value, 7, after running the fuzzer with ASAN quite a bit. 7 may be too low, but iirc I saw OOMs with a limit of 15.
Also, unlike the case with deserialization, I can't return Null and essentially force the call stack from LLVMFuzzerTestOneInput to unwind here. Unlike this case, this change could effect the behavior of the program in a way that fuzzing would miss bugs or find spurious ones. Maybe I should leave this change out until we see it's necessary. WDYT?

Reply:
I'm fine with starting with an conservatively low limit at first and then bumping it up if we think it will expand coverage